### PR TITLE
Stop error clashing with feedback message

### PR DESCRIPTION
**Pull request by Spudds**
**What does this PR add/fix/improve/...**
Stops showing an error when trying to warn GearBot as it should only display the give feedback message.

Discord_6DMsfnJFY0

### CODE REVIEW

File: GearBot/Cogs/Infractions.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ async def _warn(ctx, target, *, reason, message=True):
     async def warn(self, ctx: commands.Context, member: discord.Member, *, reason: Reason):
         """warn_help"""
         # don't allow warning GearBot, get some feedback about issues instead
-        if member.id == self.bot.user.id:
+        if member.id = self.bot.user.id:
```
Comment (AEnterprise) [inline GearBot/Cogs/Infractions.py:41]: this will not work

also isn't it just easier to just add a return to this is block as well?

Comment (Spudds) [reply GearBot/Cogs/Infractions.py:41]: Wouldn't adding a return underneath that block just get rid of the feedback thing entirely? Apologies, just a little confused.

Comment (Spudds) [reply GearBot/Cogs/Infractions.py:41]: Oh I see. Yeh, that does seem like a much more logical way of doing it. I'll add it as soon as I can.

**Comment by Spudds** [comment]
Oh whoops, didn't even mean to change that bit in the first place.

File: GearBot/Cogs/Infractions.py [hunk 2]
```diff
@@ -49,12 +49,13 @@ async def yes():
...
             message = MessageUtils.assemble(ctx, "THINK", "warn_to_feedback")
             await Confirmation.confirm(ctx, message, on_yes=yes)
+            return 
         
         if member.discriminator == '0000':
             await MessageUtils.send_to(ctx, 'NO', 'cant_warn_system_user')
             return
                         
-        if member.bot:
+        if member.bot and member.id != self.bot.user.id:
```
Comment (AEnterprise) [inline GearBot/Cogs/Infractions.py:58]: no longer needed

**Pull request merged**